db/seeds.rb

```diff
@@ -1,7 +1,18 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+user = User.create email: 'developers@example.com'
+
+project = user.projects.create! user: user, name: 'Regressor project'
+
+result = project.results.create! tag: 'master'
+result.queries.create! statement: 'SELECT * FROM users;'
+result.queries.create! statement: 'SELECT * FROM projects;'
+
+result = project.results.create! tag: 'master'
+result.queries.create! statement: 'SELECT * FROM users;'
+result.queries.create! statement: 'SELECT * FROM projects;'
+result.queries.create! statement: 'SELECT * FROM queries;'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.